### ISSUE TITLE
W2030 Valid Values for Retention in Days

### ISSUE THREAD

**Issue by sayhimatt** [aws-cloudformation/cfn-lint#2442]
### CloudFormation Lint Version

cfn-lint 0.61.3

### What operating system are you using?

Windows

### Describe the bug

![image](https://user-images.githubusercontent.com/25750611/197881654-eccdd353-4f7a-4d71-9a01-b4c46e9ae8dc.png)
![image](https://user-images.githubusercontent.com/25750611/197881715-60b4c6b9-7fe2-4441-962c-d048b0d8b149.png)
Being used for RetentionInDays attribute in a LogGroup resource. [AWS cites](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.htm) the possible values are:
1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 2192, 2557, 2922, 3288, and 3653.

### Expected behavior

It should be complicit with the provided values.

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Description: Example
Metadata:
  "AWS::CloudFormation::Interface":
    ParameterGroups:
      - Label:
          default: Configuration
        Parameters:
          - Application
          - Environment
          - LogRetentionPeriod
Parameters:
  Application:
    Description: Name of Application
    Type: String
    Default: test
  Environment:
    Type: String
    Default: development
    AllowedValues:
      - development
  LogRetentionPeriod:
    Type: String
    Description: Number of days to hold a log entry in CloudWatch
    AllowedValues: ["1", "3", "5", "7", "14", "30", "60", "90", "120", "150", "180", "365", "400", "545", "731", "1827", "2192", "2557", "2922", "3288", "3653"]
Resources:
  AssignerLogGroup:
    Type: "AWS::Logs::LogGroup"
    DeletionPolicy: Delete
    UpdateReplacePolicy: Delete
    Properties:
      RetentionInDays: !Ref LogRetentionPeriod
      LogGroupName:
        !Sub
          - /aws/lambda/${LambdaName}
          - LambdaName: !Join ["-", [!Ref Application, !Ref Environment, "assigner"]]

```

**Comment by kddejong** [comment]
Should be fixed with pr #2372.

Should be included in release [v0.65.1](https://github.com/aws-cloudformation/cfn-lint/releases/tag/v0.65.1) and later 

Upgrade cfn-lint and try again.
